features/api-token.feature

```diff
@@ -35,9 +35,8 @@ Feature: User API Token
         Then their token will have that new label
         And the token's 'last used' property will not be updated
 
-    @ignore
     Scenario: Revoke API Token
-        And owns an existing API token
-        When "calvin" revokes the token
+        Given "calvin" owns an existing API token named "tiger"
+        When they revoke the token
```

Comment:
Left it as `they` in `authorization.feature` since there's a mix in there, but updated the `api-token.feature` to use he since there's no need to be overly general there.